index.html

```diff
@@ -12,7 +12,13 @@
     <div id="screen">
         <div id="game_box">
             <div id="game_screen">
-                <h2>JavaScript Tetris</h2>
+              <div id="container">
+
+           <p><a href="https://github.com/DeltaCoderr/Tetris-Game">
```

Comment:
There really isn't a need to make that `<a href`. Perhaps you could make that as a button somewhere with the Github Logo?

Reply:
I've changed it to a "#" for now. Once you accept the pull request, you can add the website link to the "#" (similar to the anchors the modern websites have). For the github button, I've created one which is located at the top right.